scripts/globals/mobskills.lua

```diff
@@ -244,7 +230,7 @@ xi.mobskills.mobPhysicalMove = function(mob, target, skill, numHits, accMod, ftp
 
     while hitsdone < numHits do
         local isCritical = false
-        if (math.random(1, 100)) <= hitrate then --it hit
+        if math.random() <= hitrate then --it hit
```

Comment:
We already stablished math.randoms need a min and a max

Reply:
every other use of `hitrate` in lua uses frand() (which is not ambiguous on what it does because it has no inputs, the issue lied within sol not being able to tell the difference between FP/integer)